README.md

```diff
@@ -5,14 +5,14 @@ Service for monitoring of latest updates to upstream packages
 ### Configuration
 Release monitor is configurable by following environment variables.
 
-NPM_URL - URL of the NPM registry (must contain the protocol, e.g. `https://`) 
+`NPM_URL` - URL of the NPM registry (must contain the protocol, e.g. `https://`) 
 
-PYPI_URL - URL of the PyPi registry (again starting with `https://`)
+`PYPI_URL` - URL of the PyPi registry (again starting with `https://`)
  
-ENABLE_SCHEDULING - If enabled, notifications about new packages will be sent via selinon into the
+`ENABLE_SCHEDULING` - If enabled, notifications about new packages will be sent via selinon into the
 ingestion pipeline. Takes either `true` or `yes`.
   
-SLEEP_INTERVAL - Interval between fetching latest RSS feeds from registries (in minutes).
+`SLEEP_INTERVAL` - Interval between fetching latest RSS feeds from registries (in minutes).
```

Comment:
Unrelated to this PR, but is there a default sleep interval? If yes, it would be worth mentioning it here (IMO).